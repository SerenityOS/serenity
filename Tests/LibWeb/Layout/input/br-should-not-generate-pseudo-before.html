<!doctype html><style>
    body ::before { content: "+"; }
</style><body><p>P should generate a ::before pseudo.</p><br>BR should not!
