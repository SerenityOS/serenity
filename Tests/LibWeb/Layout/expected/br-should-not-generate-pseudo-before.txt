Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x83 [BFC] children: not-inline
    BlockContainer <body> at (8,16) content-size 784x67 children: not-inline
      BlockContainer <p> at (8,16) content-size 784x17 children: inline
        frag 0 from TextNode start: 0, length: 36, rect: [19,16 300x17] baseline: 13.296875
            "P should generate a ::before pseudo."
        InlineNode <(anonymous)>
          frag 0 from TextNode start: 0, length: 1, rect: [8,16 10.625x17] baseline: 13.296875
              "+"
          TextNode <#text>
        TextNode <#text>
      BlockContainer <(anonymous)> at (8,49) content-size 784x34 children: inline
        frag 0 from TextNode start: 0, length: 14, rect: [8,66 120.578125x17] baseline: 13.296875
            "BR should not!"
        BreakNode <br>
        TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x83]
    PaintableWithLines (BlockContainer<BODY>) [8,16 784x67]
      PaintableWithLines (BlockContainer<P>) [8,16 784x17]
        InlinePaintable (InlineNode(anonymous))
          TextPaintable (TextNode<#text>)
        TextPaintable (TextNode<#text>)
      PaintableWithLines (BlockContainer(anonymous)) [8,49 784x34]
        TextPaintable (TextNode<#text>)
